zpush/kweb.cfg

```diff
@@ -0,0 +1,23 @@
+:9080 {
+	log stdout
+	errors stderr
+
+	# healthcheck
+	status 200 /status
+
+	rewrite /Microsoft-Server-ActiveSync /Microsoft-Server-ActiveSync/index.php
+	fastcgi2 /Microsoft-Server-ActiveSync /run/php/php7.0-fpm.sock php {
```

Comment:
Do we really use php7.0 here - if so its time to update the base platform since PHP 7.0 is EOL since a couple of years.

Reply:
Imo we should run with PHP 7.4 (or maybe 7.3 but that will be EOL this december as well).

Reply:
No, so far this is just a simple copy of the example you provided. The containers currently use php 7.3 (which is the default provided by Debian 10)

Reply:
Ah ok